Shared/Samples/Change map view background/ChangeMapViewBackgroundView.Model.swift

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import SwiftUI
+import ArcGIS
+
+extension ChangeMapViewBackgroundView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A map with a single feature layer representing world time zones.
+        let map: Map = {
+            let featureServiceTable = ServiceFeatureTable(item: .worldTimeZones)
+            let featureLayer = FeatureLayer(featureTable: featureServiceTable)
+
+            let map = Map()
+            map.addOperationalLayer(featureLayer)
+            return map
```

Comment:
No need to change, but like to hear your thought.

Since iOS is the only platform that has this sample, we can make the layer as the baseLayer of the map, without contradicting the README steps. The old sample adds the tiled layer as the base layer.

I'm expecting this sample to be the entry level one that will be viewed by lots of first time users. Do you think it may confuse the new user by doing above? They may ask question like "what is a feature layer", "what layer can be used to create a map", etc.

```swift
return Map(basemap: .init(baseLayer: featureLayer))
```

Reply:
I'd prefer to use a map service/tiled layer as the basemap. I've asked in the Samples Slack channel for suggestions.

Reply:
I reverted to the original code.